src/store.ts

```diff
@@ -120,8 +137,11 @@ const store: TypedVuexStore = new TypedVuexStore({
     setOnboardingData(state: VuexStoreState, onboardingData: AppOnboardingData) {
       state.onboardingData = onboardingData;
     },
+    setOrderByNewest(state: VuexStoreState, orderByNewest: boolean) {
+      state.semesters = sorted(state.semesters, (state.orderByNewest = orderByNewest));
```

Comment:
I think you should just do `state.orderByNewest = orderByNewest` on a separate line to increase readability